airflow/providers/google/marketing_platform/hooks/analytics.py

```diff
@@ -0,0 +1,88 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+from googleapiclient.discovery import Resource, build
+
+from airflow.providers.google.cloud.hooks.base import CloudBaseHook
+
+
+class GoogleAnalyticsHook(CloudBaseHook):
+    """
+    Hook for Google Analytics 360.
+    """
+
+    def __init__(
+        self,
+        api_version: str = "v3",
+        gcp_connection_id: str = "google cloud default",
+        *args,
+        **kwargs
+    ):
+        super().__init__(*args, **kwargs)
+        self.api_version = api_version
+        self.gcp_connection_is = gcp_connection_id
+        self._conn = None
+
+    def get_conn(self) -> Resource:
+        """
+        Retrieves connection to Google Analytics 360.
+        """
+        if not self._conn:
+            http_authorized = self._authorize()
+            self._conn = build(
+                "analytics",
+                self.api_version,
+                http=http_authorized,
+                cache_discovery=False,
+            )
+        return self._conn
+
+    # pylint: disable=unused-argument
```

Comment:
Can you explain why we need this?

Reply:
I had to add this option because of pylint.
This methods `def list_accounts(self, max_results: int, start_index: int)` has two args, but none of them is used in method itself. Args are used in operator. 
AFAIK Analytics API method supports args: max_results and start_index but I cannot pass them into this request `analytics.management().accounts().list().execute()`. I have tried but it failed. 
I am open for the suggestions if you have any.

Reply:
I think we should remove those args if we are not using them. Otherwise, the function interface is misleading. We can add those args once we get info how to put them in API call :)

Reply:
I agree, it is confusing. I will remove these args.